cmd/controller-manager/app/options/options.go

```diff
@@ -42,8 +42,35 @@ import (
 // GenericControllerManagerOptions is the common structure for a controller manager. It works with NewGenericControllerManagerOptions
 // and AddDefaultControllerFlags to create the common components of kube-controller-manager and cloud-controller-manager.
 type GenericControllerManagerOptions struct {
-	// TODO: turn ComponentConfig into modular option structs. This is not generic.
-	ComponentConfig componentconfig.KubeControllerManagerConfiguration
+	GenericComponent                 *GenericComponentConfigOptions
```

Comment:
Can we call this `Generic`?

Reply:
Avoiding cite like`Generic.Generic`, because `CloudControllerManagerOptions` struct have `Generic` parameter. keep seems more better?